Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -884,6 +885,74 @@ namespace O3DE::ProjectManager
         }
     }
 
+    AZ::Outcome<GemInfo> PythonBindings::EditGem(const GemInfo& oldGemInfo, const GemInfo& newGemInfo)
+    {
+        using namespace pybind11::literals;
+
+        QStringList newList;
+
+        for(auto feature : newGemInfo.m_features)
+        {
+            if(!oldGemInfo.m_features.contains(feature))
+            {
+                //this must have been added by the user
+                newList << feature;
+            }
+        }
+
+        QStringList removeList;
+
+        for (auto feature : oldGemInfo.m_features)
+        {
+            if(!newGemInfo.m_features.contains(feature))
+            {
+                //this must have been removed by the user
+                removeList << feature;
+            }
+        }
+
+        GemInfo gemInfoResult;
+        bool result = ExecuteWithLock(
+            [&]
+            {
+                auto gemPath = QString_To_Py_Path(oldGemInfo.m_path);
+
+                auto editGemResult = m_gemProperties.attr("edit_gem_props")(
+                    "gem_path"_a = gemPath,
+                    "gem_name"_a = QString_To_Py_String(oldGemInfo.m_name),
+                    "new_name"_a = QString_To_Py_String(newGemInfo.m_name),
+                    "new_display"_a = QString_To_Py_String(newGemInfo.m_displayName),
+                    "new_origin"_a = QString_To_Py_String(newGemInfo.m_origin),
+                    "new_type"_a = pybind11::none(), //For the Project manager, it does not make sense to change the type of a gem once it's created
+                    "new_summary"_a = QString_To_Py_String(newGemInfo.m_summary),
+                    "new_icon"_a = QString_To_Py_String(newGemInfo.m_iconPath),
+                    "new_requirements"_a = QString_To_Py_String(newGemInfo.m_requirement),
+                    "new_documentation_url"_a = QString_To_Py_String(newGemInfo.m_documentationLink),
+                    "new_license"_a = QString_To_Py_String(newGemInfo.m_licenseText),
+                    "new_license_url"_a = QString_To_Py_String(newGemInfo.m_licenseLink),
+                    "new_tags"_a = QString_To_Py_String(newList.join(" ")),  //the python code seems to interpret these lists as space separated
+                    "remove_tags"_a = QString_To_Py_String(removeList.join(" ")),
+                    "replace_tags"_a = pybind11::none())
```

Comment:
you can omit unused/unnecessary arguments
```suggestion
                auto editGemResult = m_gemProperties.attr("edit_gem_props")(
                    "gem_path"_a = gemPath,
                    "gem_name"_a = QString_To_Py_String(oldGemInfo.m_name),
                    "new_name"_a = QString_To_Py_String(newGemInfo.m_name),
                    "new_display"_a = QString_To_Py_String(newGemInfo.m_displayName),
                    "new_origin"_a = QString_To_Py_String(newGemInfo.m_origin),
                    "new_summary"_a = QString_To_Py_String(newGemInfo.m_summary),
                    "new_icon"_a = QString_To_Py_String(newGemInfo.m_iconPath),
                    "new_requirements"_a = QString_To_Py_String(newGemInfo.m_requirement),
                    "new_documentation_url"_a = QString_To_Py_String(newGemInfo.m_documentationLink),
                    "new_license"_a = QString_To_Py_String(newGemInfo.m_licenseText),
                    "new_license_url"_a = QString_To_Py_String(newGemInfo.m_licenseLink),
                    "new_tags"_a = QString_To_Py_String(newList.join(" ")),  //the python code interprets these lists as space separated
                    "remove_tags"_a = QString_To_Py_String(removeList.join(" ")),
```

Also, you could convert the tags to a list like how it is done in UpdateProject() - might warrant a helper or macro to convert `QStringList` to `pybind11::list`